Criterion/Main/Options.hs

```diff
@@ -50,7 +50,11 @@ data MatchType = Prefix
                  -- ^ Match by prefix. For example, a prefix of
                  -- @\"foo\"@ will match @\"foobar\"@.
                | Glob
-                 -- ^ Match by Unix-style glob pattern.
+                 -- ^ Match by Unix-style glob pattern. When using this match
+                 -- type benchmark names are treated as if they were
```

Comment:
I would use a comma to separate "type" and "benchmark" here—it reads strangely without it.

Reply:
Indeed. Added a comma.